package.json

```diff
@@ -47,7 +47,7 @@
   },
   "jam": {
     "dependencies": {
```

Comment:
Not that `jam` supports peerDependencies. If you add that, it should b a level higher, not inside the `jam` property.